frontend/packages/pipelines-plugin/locales/en/pipelines-plugin.json

```diff
@@ -176,6 +177,7 @@
   "No When expressions are associated with this task": "No When expressions are associated with this task",
   "Add When Expression": "Add When Expression",
   "Select workspace...": "Select workspace...",
+  "{{workspaceName}} (deleted)": "{{workspaceName}} (deleted)",
```

Comment:
Wonder if we can reduce these down to `{{itemName}} (deleted)` instead... that way we use one for both. Translations are always going to be the same as both of these are nouns (in English) and thus should translate equally to all languages the same way.

Reply:
This will not be needed (if you agree with my other changes), but other translations will be.